crates/bevy_core_pipeline/src/tonemapping/node.rs

```diff
@@ -2,72 +2,61 @@ use std::sync::Mutex;
 
 use crate::tonemapping::{TonemappingLuts, TonemappingPipeline, ViewTonemappingPipeline};
 
-use bevy_ecs::prelude::*;
-use bevy_ecs::query::QueryState;
+use bevy_ecs::system::lifetimeless::Read;
+use bevy_ecs::{prelude::*, query::ROQueryItem};
 use bevy_render::{
     render_asset::RenderAssets,
-    render_graph::{Node, NodeRunError, RenderGraphContext},
+    render_graph::{NodeRunError, RenderGraphContext, ViewNode},
     render_resource::{
         BindGroup, BindGroupDescriptor, BindGroupEntry, BindingResource, BufferId, LoadOp,
         Operations, PipelineCache, RenderPassColorAttachment, RenderPassDescriptor,
         SamplerDescriptor, TextureViewId,
     },
     renderer::RenderContext,
     texture::Image,
-    view::{ExtractedView, ViewTarget, ViewUniformOffset, ViewUniforms},
+    view::{ViewTarget, ViewUniformOffset, ViewUniforms},
 };
 
 use super::{get_lut_bindings, Tonemapping};
 
 pub struct TonemappingNode {
```

Comment:
It should work with `derive(Default)` since `Option<T>` implements `Default` regardless of T\[[1]\]. Also `Mutex<T>`  does for `T: Default`\[[2]\]. Also just tested it myself, it does compile.

[1]: https://doc.rust-lang.org/stable/std/option/enum.Option.html#impl-Default-for-Option%3CT%3E
[2]: https://doc.rust-lang.org/stable/std/sync/struct.Mutex.html#impl-Default-for-Mutex%3CT%3E

```suggestion
#[derive(Default)]
pub struct TonemappingNode {
```

Reply:
Oh yeah, it does, not sure what was wrong last time then